.github/workflows/dart.yml

```diff
@@ -0,0 +1,284 @@
+# Created with package:mono_repo v3.4.7
+name: Dart CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - github-action
+  pull_request:
+    branches:
+      - master
+      - develop
+defaults:
+  run:
+    shell: bash
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  job_001:
```

Comment:
I don't have experience with actions, but this name001 looks like a bad thing. Can we use a more semantic name? :)

Reply:
The workflow is auto generated by mono_repo package using definition in mono_pkg.yaml files :)

Reply:
Oh, okay!